assembly/src/docs/10_INSTALL_WITH_GRAALVM.md

```diff
@@ -22,9 +22,16 @@ Example, download for Linux:
 $ wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-19.3.0/graalvm-ce-java8-linux-amd64-19.3.0.tar.gz
 $ tar -xf graalvm-ce-java8-linux-amd64-19.3.0.tar.gz
 ```
-
 The Java binary will be found in the `graalvm-ce-java{JDK_VERSION}-19.3.0` directory. This directory is used as the JAVA_HOME (See step 2).
 
+
+For OSX:
+```bash
+$ wget https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-19.3.0/graalvm-ce-java11-darwin-amd64-19.3.0.tar.gz
+```
+then untar it to `/Library/Java/JavaVirtualMachines/`
+
```

Comment:
Is this path `/Library/Java/JavaVirtualMachines/` needed? It could be any directory, couldn't it?

Reply:
It definitely can, but it is better to put it there, since it is much more "discoverable", for IDE as well.